implementations/rust/ockam/ockam_app_lib/src/incoming_services/state.rs

```diff
@@ -233,6 +261,16 @@ impl IncomingService {
         self.port = None;
     }
 
+    /// True when the service is marked as removed
+    pub fn removed(&self) -> bool {
```

Comment:
I would also rename this to `is_marked_to_be_removed` to keep it consistent with the other comment.